python/interpret_community/widget/explanation_dashboard_input.py

```diff
@@ -111,7 +112,9 @@ class and for the regression case a method of predict() returning the prediction
             if row_length > 100000:
                 raise ValueError("Exceeds maximum number of rows for visualization (100000)")
             if feature_length > 1000:
-                raise ValueError("Exceeds maximum number of features for visualization (1000)")
+                raise ValueError("Exceeds maximum number of features for visualization (1000). Please regenerate the"
+                                 " explanation using fewer features or initialize the dashboard without passing a"
+                                 " dataset.")
             self.dashboard_input[ExplanationDashboardInterface.TRAINING_DATA] = list_dataset
             self.dashboard_input[ExplanationDashboardInterface.IS_CLASSIFIER] = self._is_classifier
 
```

Comment:
at line 128 below, we may want to check the local explanation conforms to the bounds, in the case that only the local explanation is passed in w/o a dataset?